flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/AbstractHDFSWriter.java

```diff
@@ -237,14 +237,14 @@ private Method reflectHflushOrSync(FSDataOutputStream os) {
       try {
         m = fsDataOutputStreamClass.getMethod("hflush");
       } catch (NoSuchMethodException ex) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)